templates/csi-daemonset.yaml

```diff
@@ -44,7 +44,7 @@ spec:
             - name: providervol
               mountPath: "/provider"
             - name: mountpoint-dir
-              mountPath: /var/lib/kubelet/pods
+              mountPath: {{ .Values.csi.daemonSet.kubeletRootDir }}/pods
```

Comment:
This seems fine as long as the csi-provider's `kubeletRootDir` is configured the  same. I don't see the need to ever change the default value however, since this would entail deploying the worker nodes in a nonstandard way.